common.cpp

```diff
@@ -20,6 +20,12 @@ limitations under the License.
 #include <chrono>
 
 #include "common.h"
+#include <windows.h>

```

Comment:
These includes and the `FindProcessId` function below are Windows-specific, but TinyInst is multiplatform. Merging as is would cause build to break on other platforms (as you can see in the status of the pull request) Any Windows-specific code must be wrapped inside
`#if defined(WIN32) || defined(_WIN32) || defined(__WIN32)`